stack.yaml

```diff
@@ -7,14 +7,12 @@ packages:
   - aur-security/
 
 extra-deps:
-  - algebraic-graphs-0.2
-  - compactable-0.1.2.2
-  - generic-lens-1.0.0.2
-  - language-bash-0.8.0
-  - megaparsec-7.0.1
-  - mwc-random-0.14.0.0
-  - non-empty-containers-0.1.2.0
-  - paths-0.2.0.0
-  - pretty-simple-2.2.0.0
-  - throttled-1.1.0
-  - versions-3.5.0
+  - compactable-0.1.2.2@sha256:38a8f3a1089794a3d1daa50730abc98ccafaedad17cf68ffca09b271c165ff52
+  - language-bash-0.8.0@sha256:2a7b0498c029f0e7c30d3b273897e32036a629ff9ffa5c92ec2f3a51f46af86f
+  # - non-empty-containers-0.1.2.0@sha256:8e12edba9288f8c5e53c8ad4003dace546fcc5a7e9828a88b614c3049924a13a
+  - paths-0.2.0.0@sha256:85e77985968860365cdb2074515b339863344ca0d28baa879baf4105f93e87cc
+  - throttled-1.1.0@sha256:a8fac12213a23fa77badfddfb5d80139f6359b6775d661ba245cf2fc1f029bf6
+  - git: https://github.com/andrewthad/non-empty-containers
+    commit: de153be478e92b88bea88a458598975cfb9c0bba
+
+allow-newer: true
```

Comment:
For `compactable`. See https://gitlab.com/fresheyeball/Compactable/merge_requests/4